src/components/inputs-dropdowns/Input.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useRef } from 'react';
+import './inputs.css'
+
+export interface InputProps {
+    theme?: 'light' | 'dark'; // Define theme prop
+    type?: 'primary' | 'danger';
+    size?: 'desktop' | 'mobile';
+    suggestions?: string[];
+    onChange?: (value: string) => void;
+    
+}
+
+export const Input: React.FC<InputProps> = ({
```

Comment:
I think it should also accept all of `<input>`'s props and pass it through to `<input>` so a form can set `type='email'`, `autoComplete='off'`, `name='password'`, etc. I forget how to do this in React, it's probably something like `InputProps & React.HTMLProps<HTMLInputElement>` or smth, then you can do `<input {...props} etc />`